tracer/src/Datadog.Trace/ClrProfiler/ManualTracer.cs

```diff
@@ -17,6 +21,36 @@ internal ManualTracer(IAutomaticTracer parent)
             _parent.Register(this);
         }
 
+        IScope IDistributedTracer.GetActiveScope()
+        {
+            var activeTrace = _parent.GetDistributedTrace();
+
+            if (activeTrace is SpanContext)
+            {
+                // This is a local trace, no need to mock anything
+                return null;
+            }
+
+            // We don't own the active trace, get the scope from the parent and mock it
+            var activeScope = _parent.GetActiveScope();
+
+            if (activeScope is null)
+            {
+                Log.Warning("Parent tracer owns the active trace, yet the scope is null");
```

Comment:
Couldn't it also be that there _is_ no active scope at this point? 🤔  if there's no local trace, and no automatic trace?

Reply:
Yes you're right. I tested this scenario and indeed, when both ManualTracer and AutomaticTracer are activated we will hit this case whenever there is no scope. I've removed this warning so that we don't pollute the log file when this is expected behavior. Changed in 58be7bb76b0d89e569a01c8ae888ba1fcbc8ddef